routers/router.go

```diff
@@ -12,4 +12,5 @@ func init() {
 	web.Router("/register", &controllers.Registration{}, "get:New;post:Create")
 	web.Router("/login", &controllers.Session{}, "get:New;post:Create")
 	web.Router("/logout", &controllers.Session{}, "get:Delete")
+	web.Router("/keyword/:id:int", &controllers.KeywordController{}, "get:Show")
```

Comment:
@junan what's the benefit of this option `:int`?
Will it reject our request or we still have to handle it? 🤔

Reply:
It will only accept `int` value in the url. We don't need to handle it, Beego will show its default not found page if you visit`/keyword/some-string`